spec/open_api_spec.rb

```diff
@@ -45,10 +43,13 @@
     end
 
     context "customizable route prefixes" do
+      let(:path_prefix) { random_path_part }
+      let(:app_name)    { random_path }
```

Comment:
I just saw this, I remember running into an issue on Topology (I think?) that used this similar stuff, apparently "Fabergé" was a word in the dictionary on travis, and ended up breaking the URL encoding or something haha. Just something to look out for if we get a random spec failure :)